docs/spec/spec.md

```diff
@@ -1760,6 +1774,45 @@ service on new graphql:Listener(9090) {
 }
 ```
 
+#### 7.1.9 Operation Cache Configurations
+
+The `cacheConfig` field is used to provide the GraphQL operation cache configuration to enable the caching for `query` operations.
+
+###### Example: Operation Cache Configurations
+
+```ballerina
+@graphql:ServiceConfig {
+    cacheConfig: {
+        enabled: true,
+        maxAge: 60,
+        maxSize: 120
```

Comment:
These are all default values. Shall we use other values? We may move the enable cache with default values to the top, and put an example of overriding default values next.